packages/node/test/fixtures/07-content-length/probe.js

```diff
@@ -1,13 +0,0 @@
-const { strictEqual } = require('assert');
-
-async function test3({ deploymentUrl, fetch, randomness }) {
-  const bodyMustBe = `${randomness}:content-length`;
-  const resp = await fetch(`https://${deploymentUrl}/test3.js`);
-  strictEqual(resp.status, 401);
-  strictEqual(await resp.text(), bodyMustBe);
-  strictEqual(resp.headers.get('content-length'), String(bodyMustBe.length));
```

Comment:
This test was testing behavior that was non-standard (Node does not send `content-length` by default) and was a specific behavior to Vercel. Recently, the node-bridge was updated to be in streaming mode by default, so this behavior changed to not include `content-length` by default.

I deleted the test because it was not testing anything meaningful at this point.